go.mod

```diff
@@ -1,6 +1,6 @@
 module gonum.org/v1/gonum
 
 require (
-	golang.org/x/exp v0.0.0-20180321215751-8460e604b9de
-	golang.org/x/tools v0.0.0-20180525024113-a5b4c53f6e8b
+	golang.org/x/exp v0.0.0-20190125153040-c74c464bbbf2
```

Comment:
Is the plan to manually update these as necessary?

Reply:
This is done by `go get` in module mode.

Reply:
We should probably have a policy on when it's appropriate to bump a dependency's version. I don't know if there are guidelines anywhere.